testsuite/shared/src/main/java/org/jboss/as/test/integration/management/cli/AcmeMockServerBuilder.java

```diff
@@ -0,0 +1,560 @@
+        final String FINALIZE_URL = "/acme/finalize/398/186";
+
+        final String FINALIZE_RESPONSE_BODY = "{" + System.lineSeparator() +
+                "  \"status\": \"valid\"," + System.lineSeparator() +
+                "  \"expires\": \"2018-05-04T18:27:35Z\"," + System.lineSeparator() +
```

Comment:
It is valid to use already expired response?
Does it check the time of expiring?

If yes, it doesn't good to use current time + one day to expire time?

Reply:
No it does not check the expiration date since it is only a mock server which is not able to process the answers as normal ACME server would. It just returns pregenerated strings. If we would change it dynamically then the expected requests would need to be changed dynamically as well which can be done by full ACME server not by a simple mock like this.

Reply:
I didn't mean the moc server will validate the message what he sends, but I mean if the wildfly-core server validates the time of expiring of the received message from ACME server.
If the wildfly-core server validates the whole received message from ACME server.

If the wildfly-core didn't validate the answer of ACME server, isn't it the hole?

Reply:
That is a good point, I am not sure whether the Elytron does these checks when communicating with ACME server. Maybe @fjuma can give you a better answer?

Reply:
The "expires" field here is used by the ACME server (i.e., certificate authority) only. It is a timestamp after which the ACME server will consider the proof provided by the owner of the certificate to be invalid. The mock ACME server that is being used in the tests makes use of pre-recorded messages which is why it doesn't fail even though the date is in the past.